docs/integrations/mirroring.md

```diff
@@ -55,21 +57,49 @@ An example for such a function could be:
 ```python
 def get_remote_data_command(client, args):
     parsed_args = GetRemoteDateArgs(args)
-    new_incident_data: Dict = client.get_incident_data(parsed_args.remote_incident_id, parsed_args.last_update)    new_incident_data: Dict = client.get_incident_data(parsed_args.remote_incident_id, parsed_args.last_update)
-    raw_entries: List[dict] = client.get_incident_entries(parsed_args.remote_incident_id, parsed_args.last_update)
-    parsed_entries = []
-    for entry in raw_entries:
-        parsed_entries.append({       
-            'Type': EntryType.NOTE,
-            'Contents': entry.get('contents'),
-            'ContentsFormat': EntryFormat.TEXT,
-            'Tags': ['tag1', 'tag2'],  # the list of tags to add to the entry
-            'Note': False  # boolean, True for Note, False otherwise
-        })
+    try:
+      new_incident_data: Dict = client.get_incident_data(parsed_args.remote_incident_id, parsed_args.last_update)    
+      raw_entries: List[dict] = client.get_incident_entries(parsed_args.remote_incident_id, parsed_args.last_update)
+      parsed_entries = []
+      for entry in raw_entries:
+          parsed_entries.append({       
+              'Type': EntryType.NOTE,
+              'Contents': entry.get('contents'),
+              'ContentsFormat': EntryFormat.TEXT,
+              'Tags': ['tag1', 'tag2'],  # the list of tags to add to the entry
+              'Note': False  # boolean, True for Note, False otherwise
+          })
+
+      remote_incident_id = new_incident_data['incident_id']
+      new_incident_data['id'] = remote_incident_id
+      return GetRemoteDataResponse(new_incident_data, parsed_entries)
+    except Exception as e:
+      if "Rate limit exceeded" in str(e):  # modify this according to the vendor's spesific message
+      return_error("API rate limit")
+```
+
+### get-modified-remote-data
+* GetModifiedRemoteDataArgs - this is an object created to maintain all the arguments you receive from the server in order to use this command.
+Arguments explanation:
+  - last_update - Date string representing the local time that the incident was last updated.
```

Comment:
this function runs per instance not incident. 
is setLastRun and getLastRun supported here?

Reply:
Why do we need this functionality?
last_update is always modified with the last time this command was executed